.github/workflows/pr-labeler.yml

```diff
@@ -14,3 +14,35 @@ jobs:
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           configuration-path: ".github/pr-labeler-config.yml"
+      - uses: actions-ecosystem/action-add-labels@v1.1.0
+        # only add names of Acryl Data team members here
+        if: 
+          ${{ 
+            !contains(
+              fromJson('[
+                "anshbansal",
```

Comment:
might be nicer to read this from a separate file instead of having it inlined, but not a blocker